.travis.yml

```diff
@@ -8,7 +8,7 @@ language: go
 
 go:
   - 1.8.x
-  - 1.9rc2
+  - 1.9
```

Comment:
This should be 1.9.x, see https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use.

Reply:
I tried that, but unfortunately, 1.9.x doesn't match 1.9, see https://travis-ci.org/lucas-clemente/quic-go/builds/268253712.

Reply:
Okay, LGTM then. For the future, this would have been a helpful addition to the commit msg ;)